assembly/src/main/dist/docker/build.sh

```diff
@@ -30,4 +28,4 @@ cp $ZIP ./ignore/rdf4j.zip
 # build
 docker-compose build
 
-docker tag docker_rdf4j:latest eclipse/rdf4j-workbench:${MVN_VERSION}
\ No newline at end of file
+docker tag docker_rdf4j:latest eclipse/rdf4j-workbench:${MVN_VERSION}
```

Comment:
Why are there changes to the docker build script in this PR?

Reply:
Because I needed to run up the server, and I use docker for that. Got annoyed with it taking so long, so made it faster while I was at it.

Reply:
@barthanssens can we get your expert docker eye on this? This change is not really related to the issue under development here but if you're happy that it's a decent improvement we can treat it as a "tidy as you go along" thing....

Reply:
We can move this discussion here: https://github.com/eclipse/rdf4j/pull/2375